scss/base/_global.scss

```diff
@@ -0,0 +1,60 @@
+/* ---------------------------------------------------- */
+/* ----- GLOBAL STYLES ----- */
+/* ---------------------------------------------------- */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+  scroll-behavior: smooth;
```

Comment:
Ma to wsparcie w IE i Safari?

Reply:
@bartcis

Bartek coś nie działa mi ten scroll-behavior: smooth polyfill dla Safari. IE, Edge

Zrobiłam według tego i nic: https://github.com/jonaskuske/smoothscroll-anchor-polyfill

https://dev.to/pqina/css-smooth-scrolling-4m7f

Ustawiłam wszystko, jak jest w dokumentacji nie działa…

Tutaj kod: http://www.marypieroszkiewicz.com/Chilid__Recruitment__Task/index.html

Dziękuję za pomoc! :)